src/UnderlineNav2/UnderlineNav2.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import {Meta, Story} from '@storybook/react'
+import {BaseStyles, ThemeProvider} from '..'
+import {UnderlineNav} from './index'
+import {UnderlineNavItem} from './UnderlineNavItem'
+
+const excludedControlKeys = ['sx', 'as', 'variant', 'align', 'afterSelect']
+
+export default {
+  title: 'Drafts/Components/UnderlineNav',
+  component: UnderlineNav,
+  subcomponents: {UnderlineNavItem},
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ],
+  parameters: {
+    controls: {
+      expanded: true,
+      // variant and size are developed in the first design iteration but then they are abondened.
+      // Still keeping them on the source code for future reference but they are not exposed as props.
+      exclude: excludedControlKeys
+    }
+  },
+  argTypes: {
+    'aria-label': {
+      type: {
+        name: 'string'
+      }
+    },
+    children: {
```

Comment:
When `children` got updated via storybook controls, it doesn't reflect on the UI until the page is resized 🤯  I think it is very specific to UnderlineNav as the function for laying the children out on the UI (overflowEffect) only gets triggered when the page is resized or loaded. I am not too sure how to handle this. I could try to trigger calling overflowEffect function but it seems a bit anti-pattern to me. Any ideas? @joshblack